src/modules/dashboard/components/Members/MembersSection.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useState, useCallback, ReactNode } from 'react';
+import classnames from 'classnames';
+import { FormattedMessage, defineMessages } from 'react-intl';
+
+import MembersList from '~core/MembersList';
+import LoadMoreButton from '~core/LoadMoreButton';
+import SortingRow from '~core/MembersList/SortingRow';
+import { Colony, ColonyWatcher, ColonyContributor } from '~data/index';
+import useColonyMembersSorting from '~modules/dashboard/hooks/useColonyMembersSorting';
+
+import styles from './MembersSection.css';
+
+const displayName = 'dashboard.MembersSection';
+
+const MSG = defineMessages({
+  contributorsTitle: {
+    id: 'dashboard.Members.MembersSection.contributorsTitle',
+    defaultMessage: 'Contributors',
+  },
+  watchersTitle: {
+    id: 'dashboard.Members.MembersSection.watchersTitle',
+    defaultMessage: 'Watchers',
+  },
+  watchersDescription: {
+    id: 'dashboard.Members.MembersSection.watchersDescription',
+    defaultMessage: "Members who don't currently have any reputation",
+  },
+  noMemebersFound: {
+    id: 'dashboard.Members.MembersSection.noResultsFound',
+    defaultMessage: 'No members found',
+  },
+});
+
+interface Props<U> {
+  isContributorsSection: boolean;
+  colony: Colony;
+  currentDomainId: number;
+  members: ColonyWatcher[] | ColonyContributor[];
+  canAdministerComments: boolean;
+  extraItemContent: (user: U) => ReactNode;
+}
+
+const ITEMS_PER_SECTION = 10;
```

Comment:
I think this should be a Component prop that is being passed down when called _(but also has a default value)_

Reply:
issue raised